influxdb3_write/src/catalog.rs

```diff
@@ -55,7 +56,10 @@ impl Catalog {
     /// Limit for the number of Databases that InfluxDB Edge can have
     pub(crate) const NUM_DBS_LIMIT: usize = 5;
     /// Limit for the number of columns per table that InfluxDB Edge can have
-    pub(crate) const NUM_COLUMNS_PER_TABLE_LIMIT: usize = 500;
+    ///
+    /// The user-facing limit is 500, but we have it as 501 to account for the
+    /// generated _series_id column
+    pub(crate) const NUM_COLUMNS_PER_TABLE_LIMIT: usize = 501;
```

Comment:
This change is good. Glad you caught that.

Reply:
I didn't catch it, the `limits` test did! 😄